src/main/java/org/prebid/server/analytics/model/AmpEvent.java

```diff
@@ -22,8 +23,10 @@
 
     List<String> errors;
 
+    @JsonIgnore
```

Comment:
I would argue that this is a right way to fix LogAnalyticsReporter. This model definition should be usage-agnostic, i.e.  it shouldn't be assumed that this POJO will be serialized by Jackson, its primary role - convey information from one place (Handler) to another (a number of analytics modules).
Instead of doing this I would make a change to LogAnalyticsReporter so that it logs only necessary fields out of AmpEvent and AuctionEvent

Reply:
Yeah, that was like a trick. i'll discard this and let's back to it in separate PR.